test/functional/test_framework/test_framework.py

```diff
@@ -543,10 +543,15 @@ def set_dash_test_params(self, num_nodes, masterodes_count, extra_args=None, fas
         self.llmq_threshold = 2
 
     def set_dash_dip8_activation(self, activate_after_block):
-        window = int((activate_after_block + 2) / 3)
-        threshold = int((window + 1) / 2)
+        self.dip8_activation_height = activate_after_block
         for i in range(0, self.num_nodes):
-            self.extra_args[i].append("-vbparams=dip0008:0:999999999999:%d:%d" % (window, threshold))
+            self.extra_args[i].append("-dip8params=%d" % (activate_after_block))
+
+    def activate_dip8(self):
+        self.log.info("Wait for dip0008 activation")
+        while self.nodes[0].getblockcount() < self.dip8_activation_height:
+            self.nodes[0].generate(10)
+            self.sync_blocks(self.nodes)
```

Comment:
What about smth like this? I think it might be quicker

```suggestion
        self.sync_blocks(self.nodes)
```

Reply:
Tested with `./test/functional/test_runner.py feature_llmq_is_retroactive.py feature_dip4_coinbasemerkleroots.py feature_llmq_chainlocks.py feature_llmq_dkgerrors.py feature_llmq_is_cl_conflicts.py interface_zmq_dash.py` and it seems to be slightly faster indeed (~430s vs ~400s accumulated time on avg after 3 runs for each).